src/main/java/seedu/notus/data/exception/SystemException.java

```diff
@@ -21,7 +21,8 @@
         EXCEPTION_MISSING_TITLE("Missing title!"),
 
         EXCEPTION_MISSING_TIMING_PREFIX("Please include the timing prefix."),
-        EXCEPTION_MISSING_TIMING("This event does not have a timing specified!"),
+        EXCEPTION_MISSING_START_TIMING("This event does not have a timing specified!"),
+        EXCEPTION_MISSING_END_TIMING("This event does not have a timing specified!"),
```

Comment:
If it is the exact same phrase, why do need two separate names for the variables?

Reply:
Apologies. Didn't catch this mistake. I thought the error would be clearer because end time is not necessary by default. I wanted to have the error message to be missing end timing